tap/passive_tapper.go

```diff
@@ -48,6 +49,7 @@ var snaplen = flag.Int("s", 65536, "Snap length (number of bytes max to read per
 var tstype = flag.String("timestamp_type", "", "Type of timestamps to use")
 var promisc = flag.Bool("promisc", true, "Set promiscuous mode")
 var staleTimeoutSeconds = flag.Int("staletimout", 120, "Max time in seconds to keep connections which don't transmit data")
+var pids = flag.String("pids", "", "A comma separated list of PIDs to capute their netns")
```

Comment:
Yes, the default value is an empty string, the user may pass a real value. I pass a real value during tests

Reply:
@dvdlevanon could you please give an example to `pids` value? So we can try it as well.

Reply:
PID is a number, is a number of process id running in the host. So, you may pass "23423" or "3432,423"

Reply:
@dvdlevanon I understood that it's a list of comma separated PIDs. What I'm trying to say is that; it's an **incomplete integration**. There are no tests added related to this as well. So how are going to verify the code while there is no way to execute the lines?

Reply:
You shouldn't test it at this point, and no one suppose to run it for now

The feature is still in progress and im starting its next phase now.

The reason i want to approve it now, instead of waiting till the feature is done, is to help reviewers and management around it. Making small and contained PRs help everyone.

(Its much easier for me to build one giant PR with tons of changes, but i want to help the reviewers with small and contained changes.)

The reason i'm asking for review is to make sure nothing break, that i didn't miss critical issue, typos, coding conventions etc. You may reach me in DM for getting more information about the a road map, the research, the alternatives, the motivation, or anything else behind the feature.

--

Regarding tests, there is no tests to the package at all, i would be glad to add them but its out of the scope of my task.

Reply:
OK, I've manually verified that network namespaces related code works.